app/src/main/res/layout/activity_main.xml

```diff
@@ -66,6 +66,13 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
 
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
+        android:id="@+id/fullScreenDarkOverlayView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="#80000000"
+        android:visibility="gone" />
+
```

Comment:
This was the only way I found to display the tooltip overlay in front of the bottom navigation view. Similar what we are doing for `snackbarFragment`